guides/common/modules/proc_enabling-the-installer-managed-dns-service.adoc

```diff
@@ -0,0 +1,31 @@
+[id="enabling-the-installer-managed-dns-service"]
+= Enabling the installer-managed DNS service
+
+If you do not have a DNS server available in your network, you can use the installer-managed DNS service. This feature enables you to provide a DNS service with a low maintenance effort.
+
+Perform the steps on the {Project} or {SmartProxyServer} that you want to configure to manage the DNS service for the domain.
+
+
+.Procedure
+
+. Backup the answer files:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# mkdir /root/backup-answer-files/
+# cp -p /etc/foreman-installer/scenarios.d/*-answers.yml /root/backup-answer-files/
+....
+
```

Comment:
Similar here: please drop the backup step
```suggestion
```

Reply:
removed